libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -437,6 +464,24 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "reader_wpcom_sign_in_needed";
             case READER_USER_UNAUTHORIZED:
                 return "reader_user_unauthorized";
+            case READER_POST_BOOKMAKED_FROM_CARD:
+                return "reader_post_bookmarked";
```

Comment:
We might consider taking use of `switch case fallthrough` for events with the same key as I believe it's less error prone when refactoring/fixing misstypes etc. You can suppress lint warnings as I believe this is a valid use case. Wdyt?

Reply:
Good idea! Lint is not going to get angry for fallthrough with empty statements, so it's indeed a legit case :)